keps/sig-storage/4650-stateful-set-update-claim-template/kep.yaml

```diff
@@ -0,0 +1,50 @@
+title: StatefulSet Support for Updating Volume Claim Template
+kep-number: 4650
+authors:
+  - "@huww98"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-app
```

Comment:
I think the main issues with https://github.com/kubernetes/enhancements/pull/3412 were around StatefulSet controller and its behavior in error cases. IMO it should be owned by sig-apps.

Reply:
OK, given that the changes to code should mainly happen on StatefulSet controller, and the previous attempts all put sig-apps as the owning-sig (it is weird that they are placed in the sig-storage folder)